pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["setuptools>30.3.0", "wheel", "numpy>=1.13"]
```

Comment:
I wouldn't recommend adding this yet but adding it in #915 since it's best done when removing astropy-helpers

Reply:
Will do, thanks!